packages/frontend/src/index.tsx

```diff
@@ -1,8 +1,14 @@
+import { scan } from 'react-scan';
```

Comment:
needs to be imported before React

Reply:
Better add a comment and even disable eslint rule.

Example in App.ts
```
// eslint-disable-next-line import/order
import './sentry'; // Sentry has to be initialized before anything else
```

Reply:
will do!

Reply:
done!